browser/index.ts

```diff
@@ -56,6 +57,67 @@ const getNewPdfName = () => {
   return `${os.tmpdir()}/${pdfFilename}`;
 };
 
+export const previewPdf = async (
+  url: string,
+  rhIdentity: string,
+  template: ServiceNames,
+  templateData: any
```

Comment:
I think we can use `Record<string, unknown>` here instead of any. We know it has to be an object we just don't know what shape it is.